src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/XASdkTests.cs

```diff
@@ -801,20 +807,25 @@ public void DotNetBuildXamarinForms ([Values (true, false)] bool useInterpreter)
 				"android",
 				XABuildConfig.AndroidDefaultTargetDotnetApiLevel,
 			},
+			new object[] {
+				"net8.0",
+				"android",
+				XABuildConfig.AndroidDefaultTargetDotnetApiLevel,
+			},
 
 			new object[] {
-				"net7.0",
```

Comment:
Shouldn't we also assert that net7 builds continue to work?  I'm not sure I understand this change.

Reply:
Yes, the full assertion list tests 6, 7, and 8:

https://github.com/xamarin/xamarin-android/blob/5861cc818d0c9ab968aac1dfbcf3af2f328d42e5/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/XASdkTests.cs#L799-L832

I made the `net7.0` case the same as the `net6.0` one was.